announcement/announcement-api/api/src/java/org/sakaiproject/announcement/api/AnnouncementsUserNotificationHandler.java

```diff
@@ -86,6 +87,9 @@ public Optional<List<UserNotificationData>> handleEvent(Event e) {
 
         String siteId = pathParts[3];
 
+        String addEvent = AnnouncementService.SECURE_ANNC_ADD;
```

Comment:
If they are constants, they should be outside the method, at a class level.

String ADD_EVENT = XXXX